example/rds.tf

```diff
@@ -21,7 +21,7 @@ variable "cluster_state_bucket" {
 # Make sure you restart your pods which use this RDS secret to avoid any down time.
 
 module "example_team_rds" {
-  source               = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.6"
+  source               = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.7"
```

Comment:
Shouldn't this be 5.11 rather than 5.7? Other people seem to have forgotten to bump the version in earlier changes.

Reply:
yes, you are correct.